### DISCUSSION TITLE
Command not found when try to execute "make PythonAPI" in x64 Native Tools Command Prompt for VS 2022 (admin mode)

### DISCUSSION THREAD

**Discussion by kungfusung161** [carla-simulator/carla#8341] [Q&A]
Hello everyone:
I am trying to build my first CARLA. I followed the instructions from the official site. Everything went well and I finished building the unreal portion. I am using VS 2022, make 3.81, cmake 3.30.5 and python 3.8.1. I encounter the following error when I try to execute "make Python API."

**d:\CARLA_build\CARLA>make PythonAPI
/d/CARLA_build/CARLA/Util/BuildTools/BuildUE4Plugins.bat: line 1: @REM: command not found
/d/CARLA_build/CARLA/Util/BuildTools/BuildUE4Plugins.bat: line 2: setlocal: command not found
/d/CARLA_build/CARLA/Util/BuildTools/BuildUE4Plugins.bat: line 4: rem: command not found
/d/CARLA_build/CARLA/Util/BuildTools/BuildUE4Plugins.bat: line 9: syntax error near unexpected token `('
/d/CARLA_build/CARLA/Util/BuildTools/BuildUE4Plugins.bat: line 9: `rem Print batch params (debug purpose)'
make: *** [downloadplugin] Error 2**

Has anyone encounter this same issue? I would really appreciate some pointers or suggestion. I have been stuck on this for almost 3 weeks now.

**Comment by loopsmark** [comment]
The error you’re seeing suggests that the BuildUE4Plugins.bat script is being interpreted as a shell script (like Bash) rather than a Windows batch file. This is likely because the make command is being run in a Unix-like environment, such as Git Bash or WSL, which expects shell scripts rather than batch files.

**Reply by kungfusung161** [reply]
Thank you for the respond. But how do I go around this and make sure that the make command is running the windows batch file

**Reply by loopsmark** [reply]
Run the command in a normal terminal, not in git bash or wsl

**Reply by kungfusung161** [reply]
Hummm... it didn't work for me. I tried it with the cmd but still, the same error came up. I decided to try the same command and process that had the issue on another computer. This time, it did not give me any error.
